airbyte-ci/connectors/pipelines/pipelines/dagger/actions/python/common.py

```diff
@@ -211,7 +211,6 @@ async def with_installed_python_package(
     has_pyproject_toml = await check_path_in_workdir(container, "pyproject.toml")
 
     if has_pyproject_toml:
-        container = with_poetry_cache(container, context.dagger_client)
```

Comment:
Can we comment this out instead of deleting?

Reply:
Done and I added a comment to give more context